### ISSUE TITLE
OneDrive created *-PCName.7z files for sync errors, result was complete data loss for affected folders

### ISSUE THREAD

**Issue by ScottRFrost** [stefankueng/CryptSync#7]
3 devices syncing, my laptop was offline for the weekend and tried to reconnect / sync this morning.

Every clear / unencrypted folder is now empty of files, only the folders remain!

The log file just has a bunch of this:

```
07/02/18 13:50:45 : INFO:    decrypt file \\?\C:\Users\XX\OneDrive\Secure\XX\YY\ZZ-ScottFrost-PC.7z to E:\CryptSync\XX\YY\ZZ-ScottFrost-PC.7z
07/02/18 13:50:46 : ERROR:   failed to set file time on E:\CryptSync\XX\YY\ZZ-ScottFrost-PC.7z
```

Luckily I had a backup prior to using CryptSync or I'd have a complete data loss right now!

**Comment by stefankueng** [comment]
so the unencrypted folders are empty. The encrypted ones too?
when you say "offline", does that mean the paths were not accessible? Which paths (unencrypted or encrypted)?

**Comment by ScottRFrost** [comment]
Yep, all files are gone in the unencrypted folder.  This happened on all 3 computers that shared the OneDrive + CryptSync share after it occurred on the first.

![image](https://user-images.githubusercontent.com/3866418/42197330-43bef5ae-7e50-11e8-9ef7-0f74fab1c6f7.png)

**Comment by stefankueng** [comment]
Are only the files which have a *-PCName.7z equivalent file gone? Or all the others too?
And are the *-PCName.7z files "good" files, meaning they contain proper compressed files or are they maybe zero byte files?
